src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+import racingcar.model.Cars;
+import racingcar.model.RacingGame;
+import racingcar.view.Input;
+import racingcar.view.Output;
+
+public class RacingCarController {
+    private final Input input;
+    private final Output output;
+
+
+    public RacingCarController(Input input, Output output) {
+        this.input = new Input();
+        this.output = new Output();
+    }
+
+    public void startGame() {
+        List<String> carNames = input.getCarName();
+        int tryCount = input.getRound();
+        List<Car> carList = carNames.stream()
+                .map(Car::new)
+                .collect(Collectors.toList());
```

Comment:
현재 우리는 Java 21을 사용하고 있죠? 그렇다면 Java 17부터 제공되는 Stream().toList()를 사용해보는 것이 어떨까요?

```suggestion
                .toList();
```

collect(Collectors.toList()) 는 ArrayList 를 Return합니다 

Ref:javadoc

> There are no guarantees on the type, mutability, serializability, or thread-safety of the List returned

반면에 .toList() 어떤 List 객체를 반환해줄까요? 그리고 어떤 이점이 있을까요?

Collections.toList()와 Stream().toList()에 대해 그 차이점을 이번 기회에 한 번 찾아보고 알려주세요 :)